corehq/apps/builds/models.py

```diff
@@ -132,6 +133,17 @@ def get_build(cls, version, build_number=None, latest=False):
     def all_builds(cls):
         return cls.view('builds/all', include_docs=True, reduce=False)
 
+    @classmethod
+    @quickcache([], timeout=5 * 60) #This seems to be not working.
+    def j2me_enabled_builds(cls):
+        """
+        Fetch version numbers for CommCareBuilds compatible with J2ME
+        """
+        j2me_enabled_builds = []
+        for version_build in cls.all_builds():
+            if version_build.j2me_enabled:
+                j2me_enabled_builds.append(version_build.version)
+        return j2me_enabled_builds
```

Comment:
a few other pythonic ways to write this:
list comprehension:
```python
return [build for build in cls.all_builds() if build.j2me_enabled
```
filter built in
```python
return filter(lambda build: build.j2me_enabled, cls.all_builds())
```